airflow/contrib/auth/backends/ldap_auth.py

```diff
@@ -127,63 +127,14 @@ class LdapUser(models.User):
     def __init__(self, user):
         self.user = user
         self.ldap_groups = []
-        self.similar_users = []
+        self.ldap_custom_groups = []
 
         # Load and cache superuser and data_profiler settings.
         conn = get_ldap_connection(configuration.conf.get("ldap", "bind_user"),
                                    configuration.conf.get("ldap", "bind_password"))
 
-        # superuser_filter = None
-        # data_profiler_filter = None
-        # try:
-        #     superuser_filter = configuration.conf.get("ldap", "superuser_filter")
-        # except AirflowConfigException:
-        #     pass
-
-        # if not superuser_filter:
-        #     self.superuser = False
-        #     log.debug("Missing configuration for superuser settings or empty. Skipping.")
-        # else:
-        #     self.superuser = group_contains_user(conn,
-        #                                          configuration.conf.get("ldap", "basedn"),
-        #                                          superuser_filter,
-        #                                          configuration.conf.get("ldap",
-        #                                                                 "user_name_attr"),
-        #                                          user.username)
-
-        # try:
-        #     data_profiler_filter = configuration.conf.get("ldap", "data_profiler_filter")
-        # except AirflowConfigException:
-        #     pass
-
-        # if not data_profiler_filter:
-        #     self.data_profiler = False
-        #     log.debug("Missing configuration for data profiler settings or empty. "
-        #               "Skipping.")
-        # else:
-        #     self.data_profiler = group_contains_user(
-        #         conn,
-        #         configuration.conf.get("ldap", "basedn"),
-        #         data_profiler_filter,
-        #         configuration.conf.get("ldap",
-        #                                "user_name_attr"),
-        #         user.username
-        #     )
-
-        # # Load the ldap group(s) a user belongs to
-        # try:
-        #     self.ldap_groups = groups_user(
-        #         conn,
-        #         configuration.conf.get("ldap", "basedn"),
-        #         configuration.conf.get("ldap", "user_filter"),
-        #         configuration.conf.get("ldap", "user_name_attr"),
-        #         user.username
-        #     )
-        # except AirflowConfigException:
-        #     log.debug("Missing configuration for ldap settings. Skipping")
-
-        self.similar_users = self.get_similar_users(conn)
-        log.info("Similar user are :{}".format(self.similar_users))
+        self.get_current_user_group(conn)
```

Comment:
Cette fonction ne devrait-elle pas être renommée ?

- `set` ou `resolve` ou `initialize` comme préfixe ?
- garder la notion de `custom_group` dans le nom ?

Reply:
The name has been changed to resolve_current_user_custom_group